databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -37,6 +38,16 @@ public class DatabricksConfig {
       sensitive = true)
   private String clientSecret;
 
+  @ConfigAttribute(value = "scopes", env = "DATABRICKS_SCOPES", auth = "oauth", sensitive = true)
```

Comment:
Scopes are not sensitive, are they?

Reply:
Sensitive here means that it shouldn't be put into logs, not that it is case-sensitive.

Reply:
My bad. I misunderstood. :( Updated